lib/src/lib.rs

```diff
@@ -141,7 +175,13 @@ impl Tester {
     }
 
     /// Test an Actor on a `MemoryBlockstore`.
-    pub fn test(&mut self, test_actors: &[WasmActor]) -> Result<Vec<TestResults>, Error> {
+    /// TODO: Instead of accepting `stream_results` as a channel that we then yield each result,
+    /// Should we make `test` return an iterator that yields each result by default?
```

Comment:
basically, instead of returning `Vec<TestActorResults<'a>>` return something like `SuiteResult<'a>` that impls `Iterator`, has internal acess to the machine and the list of actor tests, and everytime you call `.next()` it would `execute_message` for a given method.